app/client/components/Note/note.jsx

```diff
@@ -8,43 +8,69 @@ import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 import ShareIcon from '@mui/icons-material/Share';
 import ExitToAppIcon from '@mui/icons-material/ExitToApp';
+import axios from "axios";
+import { API_URL } from "../../next.config";
+import { useRouter } from 'next/router'
+import EditNote from "../../components/PopUps/EditNote";
 
 const MDEditor = dynamic(
     () => import("@uiw/react-md-editor"),
     { ssr: false }
 );
 
-function Note({note, deleteNote}) {
+function Note({ note, openEditNote, setOpenEditNote, space_id,setEditNote}) {
+
+    const router = useRouter();
     const [actions, setActions] = useState(false);
 
+    const deleteNote = async () => {
+        console.log(note._id)
+        const body = {
+            note_id: note._id
+        }
+        try {
+            await axios.delete(API_URL + "/note/delete", body);
+        } catch (err) {
+            console.log(err);
+        }
+    }
+
     return (
-        <Card onMouseEnter={() => setActions(true)} onMouseLeave={() => setActions(false)}>
-            <CardContent>
-                <div data-color-mode="light">
-                    <MDEditor
-                        value={note.value}
-                        onChange={() => { }}
-                        preview="preview"
-                        hideToolbar={true}
-                        height={100}
-                        previewOptions={{
-                            rehypePlugins: [[rehypeSanitize]],
-                        }}
-                    />
-                </div>
-            </CardContent>
-            <Collapse in={actions}>
-                <CardActions style={{ marginTop: "-1rem" }}>
-                    <ButtonGroup variant="outlined" aria-label="outlined button group" fullWidth style={{ padding: "8px" }}>
-                        <Button><ExitToAppIcon /></Button>
-                        <Button><EditIcon /></Button>
-                        <Button><ShareIcon /></Button>
-                        <Button onClick={deleteNote}><DeleteIcon /></Button>
-                    </ButtonGroup>
-                </CardActions>
-            </Collapse>
-        </Card>
+   
+            <Card onMouseEnter={() => setActions(true)} onMouseLeave={() => setActions(false)}>
```

Comment:
I think we should add name of the resource notes belong to somewhere here

Reply:
You are right. But right now our notes have no title. I think we should leave it as it for the moment. Also in note creation pop up title is just set to "" for the moment.